02.21.16

manitghogar opened the pull request.

manitghogar committed:
LEARN MORE PRINTING

Learnt more about format variables and joining variables together into
strings

manitghogar committed:
Printing into Format Variables

Learnt how to assign numbers, booleans and strings to four format
variables defined by one variable.

manitghogar committed:
Print variables into strings

Learn about printing variables into strings,and also about commenting
out multiple lines.

manitghogar committed:
Learn escape sequences

Combining lessons about format strings and escape sequences. Realized
that %r and %s have different outputs when combined with escape
sequences. %r shows it as it is typed, %s shows it as it is intended.

manitghogar committed:
Asking Questions and Input Fields

Learn about input fields and incorporating them into strings with
format variables.

manitghogar committed:
Prompting People

Learnt to prompt people by asking questions in a more effective way.
The prompt is incorporated into the question, and the value of the
answer is assigned to the variable.

manitghogar committed:
Learn parameters, unpacking and variables

Learnt how to combine raw_input() and argv

manitghogar committed:
Learn how to prompt and pass

Learnt to prompt users for answers and then use these values in the
final multiline string output using format activators.

arjunblj commented on 11/ex11.py:
```diff
@@ -0,0 +1,27 @@
+#prints question (argument)
+print "How old are you?" ,
+#input field for age
+age = raw_input ()
```
This runs fine, but like I mentioned, no spaces between the function name and call — `raw_input()`.

arjunblj commented:
LGTM (looks good to me)

manitghogar committed:
Learn to read files

Learnt different ways to open files in python. The better way in my
opinion is when Python prompts you to open a file of your chosing.
